includes/functions/banner.php

```diff
@@ -144,7 +144,7 @@ function zen_display_banner($action = '', $identifier = '') {
         if ($banner->fields['banners_open_new_windows'] == '1') {
           $target = ' rel="noopener" target="_blank"';
         }
-        $banner_string = '<a href="' . zen_href_link(FILENAME_REDIRECT, 'action=banner&goto=' . $banner->fields['banners_id']) . '"' . $target . '>' . zen_image(DIR_WS_IMAGES . $banner->fields['banners_image'], $banner->fields['banners_title']) . '</a>';
+        $banner_string = '<a href="' . zen_href_link(FILENAME_REDIRECT, 'action=banner&goto=' . $banner->fields['banners_id']) . '"' . $target . ' aria-label="' . $banner->fields['banners_url'] . '">' . zen_image(DIR_WS_IMAGES . $banner->fields['banners_image'], $banner->fields['banners_title']) . '</a>';
```

Comment:
Should the label be from `banners_url` as here, or `banners_title`?

Reply:
Changed - it just needs something whether it's url or title.